appinventor/components-ios/tests/Unit Tests/util/LinearViewTests.swift

```diff
@@ -73,6 +73,35 @@ class LinearViewTests: XCTestCase {
     Button1.Width = kLengthFillParent
   }
 
+  func testAutomaticParent() {
```

Comment:
Add a second test function that mimics switching between projects as we did in the office to verify the fix. This way if we make further changes and break it we will know before merging.